src/app/components/link-records/index.html

```diff
@@ -16,68 +17,68 @@
     </sky-demo-page-property>
     <sky-demo-page-property
-      propertyName="matchFields"
-      defaultValue="[]"
+      propertyName="itemTemplate"
       isOptional="true"
     >
-    Specifies the property names of the fields that can be updated when editing. Takes in an object with a <sky-code>key</sky-code> and <sky-code>label</sky-code> property. the key specifies the field name, and the label defines the text to show when matching fields. if no label is defined, the key name is displayed instead. Acceptable values: <sky-code>Observable&lt;Array&lt;any&gt;&gt;</sky-code> or <sky-code>Array&lt;any&gt;</sky-code>.
+    Specifies the <sky-code>TemplateRef</sky-code> for the item template content section.
     </sky-demo-page-property>
     <sky-demo-page-property
-      propertyName="showNewFieldValues"
-      defaultValue="true"
+      propertyName="itemTitleTemplate"
       isOptional="true"
     >
-    Specifies whether or not the user will be able to choose to update new values for a matched field that contains no current value. If false, the field will be updated automatically without input needed from the user. Acceptable values: <sky-code>true</sky-code> or <sky-code>false</sky-code>.
+    Specifies the <sky-code>TemplateRef</sky-code> for the item title template content section.
     </sky-demo-page-property>
     <sky-demo-page-property
-      propertyName="itemTemplate"
+      propertyName="keyIdSelector"
+      defaultValue="id"
       isOptional="true"
     >
-    Specifies the <sky-code>TemplateRef</sky-code> for the item template content section.
+    Specifies the key property name to use to link the original item with the matching item. Must provide <sky-code>string</sky-code>.
     </sky-demo-page-property>
     <sky-demo-page-property
-      propertyName="matchTemplate"
+      propertyName="matches"
+      defaultValue="[]"
       isOptional="true"
     >
-    Specifies the <sky-code>TemplateRef</sky-code> for the match template content section.
+    Specifies the match data for the items provided. Acceptable values: <sky-code>Observable&lt;Array&lt;SkyLinkRecordsMatchModel&gt;&gt;</sky-code> or <sky-code>Array&lt;SkyLinkRecordsMatchModel&gt;</sky-code>.
     </sky-demo-page-property>
     <sky-demo-page-property
-      propertyName="noMatchTemplate"
+      propertyName="matchFields"
+      defaultValue="[]"
       isOptional="true"
     >
-    Specifies the <sky-code>TemplateRef</sky-code> for the no match template content section.
+    Specifies the property names of the fields that can be updated when editing. Takes in an object with a <sky-code>key</sky-code> and <sky-code>label</sky-code> property. the key specifies the field name, and the label defines the text to show when matching fields. if no label is defined, the key name is displayed instead. Acceptable values: <sky-code>Observable&lt;Array&lt;any&gt;&gt;</sky-code> or <sky-code>Array&lt;any&gt;</sky-code>.
```

Comment:
Beginnings of second and third sentences need capitalization

Reply:
Fixed. Thanks!